### PR TITLE
Move ObjectSearchCreate Add New to dropdown

### PR DESCRIPTION
**Pull request by rkuykendall**
osc-v2

### CODE REVIEW

File: src/inputs/ObjectSearchCreateSearchInput.tsx [hunk 1]
```diff
@@ -153,16 +165,26 @@ class ObjectSearchCreateSearchInput extends Component<IObjectSearchProps> {
       </Antd.Select.Option>
     );
   }
+  private onChange (selectedOption: any) {
```
Comment (alexandermeyer) [inline src/inputs/ObjectSearchCreateSearchInput.tsx]: Spacing before this function

File: src/inputs/ObjectSearchCreateSearchInput.tsx [hunk 2]
```diff
@@ -153,16 +165,26 @@ class ObjectSearchCreateSearchInput extends Component<IObjectSearchProps> {
...
+    }
 
-  private onChange (value: any) {
-    const foundOption = this.options.find(option => option.id === value.key);
+    const foundOption = this.options.find(option => option.id === selectedOption.key);
     this.injected.onChange(toJS(foundOption));
   }
 
+  private onFocus () {
+    if (!this.hasOptions && !this.hasSearch) {
```
Comment (alexandermeyer) [inline src/inputs/ObjectSearchCreateSearchInput.tsx]: This `onFocus` is searching with an empty string to get a list of all possible options on initial click?

Comment (rkuykendall) [reply src/inputs/ObjectSearchCreateSearchInput.tsx]: I believe so. @kbouchard ?

Comment (kbouchard) [reply src/inputs/ObjectSearchCreateSearchInput.tsx]: ex-ac-tly

Focusing a "list" selection that would render no values at first was really confusing UX.

Comment (alexandermeyer) [reply src/inputs/ObjectSearchCreateSearchInput.tsx]: Then I would name a boolean something like that so it's clearer to the next person who reads it!

**Pull request merged**